apps/calendar/js/views/advanced_settings.js

```diff
@@ -141,7 +141,7 @@ AdvancedSettings.prototype = {
   onCreateAccount: function(event) {
     event.stopPropagation();
     event.preventDefault();
-    router.show(event.target.getAttribute('href'));
+    router.show(event.target.dataset.href);
```

Comment:
BTW, only reason why I did not convert it into a link was because of the styling and because we already had the listeners in place.